os/include/tinyara/binary_manager.h

```diff
@@ -101,6 +101,7 @@ enum binmgr_response_result_type {
 	BINMGR_INVALID_PARAM = -4,
 	BINMGR_NOT_FOUND = -5,
 	BINMGR_ALREADY_REGISTERED = -6,
+	BINMGR_ALREADY_LATEST_VERSION = -7,
```

Comment:
Grammatically, ALREADY_UPDATED is correct

Reply:
Thanks. i will fix